graphql/connection_test.go

```diff
@@ -511,9 +529,9 @@ func TestPaginateBuildFailure(t *testing.T) {
 		object.Key("id")
 		inner.FieldFunc("connection", func(ctx context.Context, args Args) ([]StructWithKey, error) {
 			return nil, nil
-		}, schemabuilder.Paginated, schemabuilder.TextFilterFields{
-			"intReturn": func() int64 { return 0 },
-		})
+		}, schemabuilder.Paginated, schemabuilder.BatchFilterField(
```

Comment:
This shouldn't work right?

Reply:
nope, I'll double check, but I'm pretty sure this is testing an error case

Reply:
yup, it returns a `"unsupported return type int64"`